### ISSUE TITLE
Myself datasets

### ISSUE THREAD

**Issue by JJJTF** [sheryl-ai/MVGCN#3]
How to use my own multiview dataset? My dataset has two views representing an object, 10 categories and 30000 objects.Similar to modelnet dataset, but smaller in number.
Thank you very much!

**Comment by sheryl-ai** [comment]
We only have 2 categories here. Randomly split samples into train and test. Then sample pairs for train and test sets, respectively. You can obtain a file with the same format as our "dti.pair.pkl". Considering you have 10 categories, splitting train and test for each category (keep the same label distribution for train and test) would be a good choice. The views are read one-by-one in our case before training (the sample size is smaller than yours). You can see the loop in line 29.  "dti.coo.pd.pkl" is 3d coordinates for each sample's brain regions. We just aggregate them along the sample axis to get an average coordinates.  If you have any further question, feel free to discuss.

**Comment by JJJTF** [comment]
Thank you very much for your prompt reply!

How should I generate my image dataset to ‘ dti.coo.pkl‘ ?
This is very important to me!

**Comment by sheryl-ai** [comment]
Yes. The 'dti.coo.pkl' needs to be generated by yourself. Basically, this is for 3D information. [n_samples x n_nodes x n_XYZ] is its dim. n_nodes in our case means the number of brain regions, n_XYZ is the associated 3D coordinates, equals to 3.

**Comment by JeenaGeorge7** [comment]
Can you please upload the input file  dti.pair.pkl
